src/components/Carousel/Carousel.tsx

```diff
@@ -145,16 +147,17 @@ export const Carousel: FC<CarouselProps> = ({
     >
       <ScrollContainer
         className={twMerge(theme.scrollContainer.base, (isDeviceMobile || !isDragging) && theme.scrollContainer.snap)}
-        draggingClassName="cursor-grab"
+        draggingClassName={'cursor-grab'}
```

Comment:
can be reverted to `"cursor-grab"`, no improvement here

Reply:
will be included in next commit